<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1055</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>331</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40,150);</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_target_elevation">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>100</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40,0);</string>
     </property>
     <property name="text">
      <string>Latitude</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40,0);</string>
     </property>
     <property name="text">
      <string>Grades</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40,0);</string>
     </property>
     <property name="text">
      <string>Longitude</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_both">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt;</string>
     </property>
     <property name="text">
      <string>curve</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_lon">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(78, 154, 6);</string>
     </property>
     <property name="text">
      <string>0.00ยบ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_lat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(78, 154, 6);</string>
     </property>
     <property name="text">
      <string>0.00ยบ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="lift_off_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt;</string>
     </property>
     <property name="text">
      <string>lift off</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_land_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt;</string>
     </property>
     <property name="text">
      <string>to land</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>871</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>871</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layaout_planner"/>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>151</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40,150);</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40,0);</string>
     </property>
     <property name="text">
      <string>Fuel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_fuel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(78, 154, 6);</string>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="plane_on_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 8pt;</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_state">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 40, 40,0);</string>
     </property>
     <property name="text">
      <string>State</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_lat_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(78, 154, 6);</string>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionUpdate_constellations">
   <property name="text">
    <string>Update catalog</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
